typed-racket-lib/typed-racket/env/global-env.rkt

```diff
@@ -8,7 +8,7 @@
          "env-utils.rkt"
```

Comment:
It seems like the code in this file is unchanged, but there are two extremely minor white space edits. Is it worth keeping these edits to muck up this file's history a little?